src/transports/mod.rs

```diff
@@ -31,3 +31,6 @@ impl From<native_tls::Error> for crate::Error {
         crate::Error::Transport(format!("{:?}", err))
     }
 }
+
+#[cfg(feature = "eip-1193")]
```

Comment:
Oh, the feature gate is already here, so an extra one in the module is definitely not needed.